src/Html2Text.php

```diff
@@ -222,6 +220,9 @@ class Html2Text
         'width' => 70,          //  Maximum width of the formatted text, in columns.
                                 //  Set this value to 0 (or less) to ignore word wrapping
                                 //  and not constrain text to a fixed-width column.
+
+        'alt_image'=>'show',    //  'show' (show the alt text in brackets)
```

Comment:
I'd suggest that this be names `images` with options:
- `none`
- `alt` (default)

This could then be expanded in future to, for example, include the path to the image if desired.